### PR TITLE
Fix Python3 issues for F26

### PR DESCRIPTION
**Pull request by baude**
## Description
Several tests failed (as reported in issues #1057) on Fedora 26.

## Related Issue Numbers
- #1057 1057
-

## Pull Request Checklist:

If your Pull request contains new features or functions, tests are required. If the PR is a bug fix and no tests exist, please consider adding some to prevent regressions.
- [ ] Unittests
- [ ] Integration Tests

### CODE REVIEW

**Comment by baude** [comment]
bot, retest this please

**Comment by baude** [comment]
@rhatdan ptal

File: tests/integration/test_system_containers_images.sh [hunk 1]
```diff
@@ -114,15 +114,16 @@ cat > test_images_size.py <<EOF
 import json
 import sys
 def make_number(x):
-    return float(x.replace("MB", "").replace("KB", ""))
+    foo = x.replace("MB", "").replace("KB", "").replace(" ","")
```
Comment (TomSweeneyRedHat) [inline tests/integration/test_system_containers_images.sh]: We don't have to worry about GB I'm assuming?

File: Atomic/containers.py [hunk 2]
```diff
@@ -120,7 +119,7 @@ def _walk(_filter_objs, _filter, _value):
 
         if not self.args.filter:
             return con_objs
-        filtered_objs = copy.deepcopy(con_objs)
+        filtered_objs = con_objs[:]
```
Comment (TomSweeneyRedHat) [inline Atomic/containers.py]: I think (testing my python knowledge here) is changing from a deep copy to a shallow copy.  The deep will make a copy of the original object, the shallow adds a pointer to the original object.  I think that's ok, but just wanted to bring this up on the off chance that you've not mulled it over.

**Comment by rhatdan** [comment]
@baude Answer Tom's question, and it LGTM.

File: Atomic/containers.py (hunk 2 above)
Comment (baude) [inline Atomic/containers.py]: Yeah, @giuseppe helped me with this one and we mulled it over ...

File: tests/integration/test_system_containers_images.sh (hunk 1 above)
Comment (baude) [inline tests/integration/test_system_containers_images.sh]: I didn't write the test (I just fix things), but evidently no.

**Comment by baude** [comment]
@rhatdan questions answered ...

**Comment by rhatdan** [comment]
@rh-atomic-bot r+ af4a75a

**Pull request closed without merging**